enterprise/static/enterprise/sass/partials/views/_data_sharing_consent.scss

```diff
@@ -78,13 +78,17 @@
   }
 }
 
+.policy-dropdown-link {
+  text-decoration: underline;
+}
+
 .failure-link {
   color: #0075B4;
   display: inline-block;
   text-decoration: underline;
   border: none;
   background-color: rgba(0, 0, 0, 0);
-  padding: 0 !important;
+  padding: 10px 15px;
```

Comment:
Just verifying this change was intentional as you removed .failure-link from anchors below

Reply:
Yeah, the `.failure-link` class was removed from the "Data Sharing Policy" link that opens the dropdown content. The class is still in use for the secondary button options (i.e., "No, take me back" and "View the data sharing policy"), and the padding change is what fixes the vertical alignment issue to match the spacing of the primary button.